src/ddmd/parse.d

```diff
@@ -21,6 +21,7 @@ import ddmd.root.filename;
 import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 import ddmd.root.rootobject;
+import ddmd.target;
```

Comment:
Apparently the Jenkins ci is broken and preventing auto merges. In any case you are making a claim that falls on deaf ears because the fix was already made available 24 hours ago.